app/controllers/veterans_controller.rb

```diff
@@ -1,6 +1,8 @@
 class VeteransController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
i think we added this when we were trying to fix an issue. if we remove this line, does everything still work?

Reply:
I think we need it to disable CSRF protection and let mobile requests hit the controller

Reply:
ah ok that makes sense @lbkchen